Optik/Classes/AlbumViewController.swift

```diff
@@ -68,6 +68,7 @@ internal final class AlbumViewController: UIViewController {
     private var viewDidAppear: Bool = false
     
     private var transitionController: TransitionController = TransitionController()
+    fileprivate var pageControl: UIPageControl?
```

Comment:
If I add pageControl as a private variable, I am not able to access it on the extension. Should I keep it fileprivate?